hack/tools/internal/update_providers/README.md

```diff
@@ -0,0 +1,35 @@
+# Update CAPI Providers with CAPI beta releases
+
+`provider_issues` is a go utility intended to open git issues on provider repos about the latest CAPI beta release.
```

Comment:
is this tool used only once per cycle, e.g when first beta release is out or we are planning to update for each beta releases (-beta.0, -beta.1 etc)?

Reply:
Currently, this tool is to be used only once per release cycle. I.e. to update providers with the release of one beta release.

Reply:
Now that I think about it, we could update the utility to update the providers for each beta release. 
But the con to that is that each provider has their own workflow, and an update for each beta release might overwhelm them.

Reply:
addressed via 756bcb44a70938a7b18f14951bf99c1658880640

Reply:
I think informing once with the first beta release is enough also and my intention with the question was: if we inform them once, we need to make it clear in the notes, i,e:
```suggestion
`provider_issues` is a go utility intended to open git issues on provider repos about the first beta minor release of CAPI.
```

Reply:
Good catch. Updated the readme.md